server/db/add_client.sh

```diff
@@ -39,7 +39,8 @@ else
     echo "  Responder Phones: $3"
     echo "  Fallback Phones: $4"
     echo "  From Phone: $5"
-    sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER --set=sslmode=require -c "INSERT INTO clients (display_name, responder_phone_numbers, fallback_phone_numbers, from_phone_number, is_displayed, is_sending_alerts, is_sending_vitals) VALUES ('$2', '$3', '$4', '$5', 't', 'f', 'f');"
+    echo "  Locationid: $6"
+    sudo PGPASSWORD=$PG_PASSWORD psql -U $PG_USER -h $PG_HOST -p $PG_PORT -d $PG_USER --set=sslmode=require -c "INSERT INTO clients (display_name, responder_phone_numbers, fallback_phone_numbers, from_phone_number, locationid, is_displayed, is_sending_alerts, is_sending_vitals) VALUES ('$2', '$3', '$4', '$5', '$6', 't', 'f', 'f');"
```

Comment:
There isn't a locationid column on the clients table, so these two lines (42, 43) should be unchanged